packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -31,6 +31,8 @@ Requires: %{?scl_prefix}rubygem(ipaddress) < 1.0
 BuildRequires: foreman-assets >= %{foreman_min_version}
 BuildRequires: foreman-plugin >= %{foreman_min_version}
 BuildRequires: %{?scl_prefix}rubygem(deface) < 2.0
+BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 2.0.2
```

Comment:
Why is this here? AFAIK it's not an actual dependency. It's mentioned as a development dependency so the tests can run, but this gem should be called `smart_proxy_ansible_core`. Refactoring that was a bit hard for me, but it'd really clear up the situation for everyone involved.

Reply:
This follows REX, tasks (and probing) pattern where some code needed to be shared. We gave up (I think) on running REX from core without proxy already so you're right, it may make more sense to be renamed, not something that I'd like to solve in this PR though. I suppose we should also extract that to a separate repo. cc @iNecas

Reply:
But it's not actually a dependency anymore. That's why I left it out in the (autogenerated) dependency list. https://github.com/theforeman/foreman_ansible/commit/e88cfd7efd7ad8abe8b37f2159190e622b79c841 removed it and only kept it as a development dependency for tests (https://github.com/theforeman/foreman_ansible/blob/e805df4ba6f4366423b369c5746a58fc662034d8/foreman_ansible.gemspec#L21-L22). I think that one doesn't make sense since it will pull the gem from rubygems rather than using the actual git sources. Will you only see see failures after you release?